src/linter/disable_directives/Parser.zig

```diff
@@ -198,8 +197,8 @@ test parse {
                 .kind = .global,
                 .span = .{ .start = 0, .end = 28 },
                 .disabled_rules = @constCast(&[_]Span{
-                    Span.new(17, 21),
-                    Span.new(21, 25),
+                    Span.new(17, 20),
```

Comment:
Maybe we should add a check that a slice taken by this `Span` still has the expected contents?